jax/interpreters/xla.py

```diff
@@ -618,17 +733,34 @@ def _get_device(device, backend):
 xla_call_p.def_impl(_xla_call_impl)
 pe.staged_out_calls.add(xla_call_p)
 
-def _xla_call_translation_rule(c, axis_env,
-                               in_nodes, name_stack, backend, name,
-                               call_jaxpr, device=None):
+def _xla_call_translation_rule(
+    c: xc.XlaBuilder,
+    axis_env,
+    in_nodes,
+    name_stack: str,
+    backend,
+    name: str,
+    call_jaxpr,
+    device: Optional[xe.Device],
+    donate: Sequence[bool],
+):
   del device  # Ignored.
+  if not all(not x for x in donate):
```

Comment:
The exception message suggests that the check should be `if any(donate)`.

Reply:
🤦 yep that would be a lot simpler 😄